osf/metadata/serializers/datacite/datacite_tree_walker.py

```diff
@@ -388,6 +391,24 @@ def _visit_related(self, parent_el):
                 datacite_relation,
             )
 
+        # Then add verified links to same relatedIdentifiers element
+        osf_item = self.basket.focus.dbmodel
+        from osf.models import AbstractNode
+
+        if isinstance(osf_item, AbstractNode):
+            gv_verified_link_list = osf_item.get_links()
+            for item in gv_verified_link_list:
+                verified_link, resource_type = item.get('target_url', None), item.get('resource_type', None)
+                if verified_link and resource_type:
+                    if verified_link and isinstance(verified_link, str) and smells_like_iri(verified_link):
+                        self.visit(related_identifiers_el, 'relatedIdentifier', text=verified_link, attrib={
+                            'relatedIdentifierType': 'URL',
+                            'relationType': 'IsReferencedBy',
+                            'resourceTypeGeneral': resource_type.title()
+                        })
+                    else:
+                        logger.warning('skipping non-URL verified link "%s"', verified_link)
```

Comment:
This would be more useful if logged to sentry. And add which node it is.

It is probably nicer if we keep a list of skipped ones and send only one sentry message after the for loop.